@here/harp-examples/src/threejs_shadows.ts

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    isLiteralDefinition,
+    Style,
+    StyleDeclaration,
+    Theme
+} from "@here/harp-datasource-protocol";
+import { isJsonExpr } from "@here/harp-datasource-protocol/lib/Expr";
+import { GeoCoordinates } from "@here/harp-geoutils";
+import { MapControls, MapControlsUI } from "@here/harp-map-controls";
+import {
+    CopyrightElementHandler,
+    MapView,
+    MapViewEventNames,
+    ThemeLoader
+} from "@here/harp-mapview";
+import { APIFormat, AuthenticationMethod, OmvDataSource } from "@here/harp-omv-datasource";
+import { assert } from "@here/harp-utils";
+import { GUI } from "dat.gui";
+import * as THREE from "three";
+import "three/examples/js/controls/TrackballControls";
+import { ShadowMapViewer } from "three/examples/jsm/utils/ShadowMapViewer";
+import { apikey, copyrightInfo } from "../config";
+
+let map: MapView;
+
+const guiOptions = {
```

Comment:
MINOR: Would be good to have some initial values that actually show shadows

Reply:
Agreed, done.